host-controller/src/main/java/org/jboss/as/domain/controller/operations/HostProcessReloadHandler.java

```diff
@@ -76,10 +76,13 @@
             .setAlternatives(ModelDescriptionConstants.USE_CURRENT_DOMAIN_CONFIG)
             .build();
 
+    protected static final AttributeDefinition TIMEOUT = new SimpleAttributeDefinitionBuilder(ModelDescriptionConstants.TIMEOUT, ModelType.INT, true)
+            .setDefaultValue(new ModelNode(0)).build();
```

Comment:
Does the suspend operation block? I don't remember off the top of my head.

If it does, then won't the two server-one suspends will finish before the master reload. If not, I see your point :)